README.md

```diff
@@ -1,48 +1,68 @@
-# protego
+# Protego
 
-Protego is a tool to permissionlessly deploy a spell which can be used to drop a particular plan in MakerDAO's Pause contract, or in extreme cases the protego contract can itself be lifted to the hat, allowing any user to permissionlessly drop any scheduled plan.
+Protego is a tool to permissionlessly deploy a spell which can be used to drop a particular plan in the `MCD_PAUSE`
+contract, or in extreme cases the `Protego` contract can itself be lifted to the hat, allowing any user to
+permissionlessly drop any scheduled plan.
+
+## Context
+
+- A `plan` is a scheduled `delegatecall` that exists in [`MCD_PAUSE`](https://github.com/makerdao/ds-pause) and can be
+  permissionlessly executed.
+- Plans in `MCD_PAUSE` are [identified by a unique 32 byte hash](https://github.com/makerdao/ds-pause/blob/master/src/pause.sol#L99).
+- A conformant spell here is one where the values returned by `action()`, `tag()`, `sig()` and `eta()` are equal to the
+  corresponding `plan`'s `usr`, `tag`, `fax` and `eta` values respectively. Bad actors could potentially manipulate the
+  return values of these functions such that they are not equal, which would result in a non-conformant spell.
 
 ## Usage
 
-### Drop an individual plan
+### Create a single drop spell
 
-Protego contains a factory to permissionlessly create a spell that is able to drop a single plan in the [pause](https://github.com/dapphub/ds-pause).
+If there is sufficient ecosystem interest to cancel (`drop`) a particular scheduled set of actions (the target `plan`),
+Protego contains a factory to permissionlessly create a spell ("drop spell"), which &ndash; upon being given the hat
+&ndash; is able to drop the targeted plan in `MCD_PAUSE`.
 
-* `function deploy(DSSpellLike _spell) external returns (address)`
+There are two possibilities:
 
-Used to deploy a single spell that can be elected to a privileged position. This requires that the `_spell` address parameter conforms to a MakerDAO conformant [spell](https://github.com/makerdao/spells-mainnet)
+#### 1. `deploy(DssSpellLike _spell)(address)`
```

Comment:
There's a case where a conforming spell might need to be dropped. For instance: some last minute legal issue that was found or a bug in the spell.

Reply:
I understand that there are conformant spells that might need to be dropped. I rather question why not only have `deploy(address _usr, bytes32 _tag, bytes memory _fax, uint256 _eta)(address)` as it can be used to drop both types of spells and more importantly does not require expertise to decide if the spell is conformant. You can have a script along the repo to fetch all plotted spells and parse required parameters for calling this function no matter if the spell is conformant or not.

Reply:
See [the comment](https://github.com/makerdao/protego/pull/2#discussion_r1600335613) for a scenario when calling `deploy(DssSpellLike _spell)(address)` could be weaponised

Reply:
Agreed, it will make it more difficult instead of easier for users.
Code was updated.